celery/backends/elasticsearch.py

```diff
@@ -38,6 +38,8 @@ class ElasticsearchBackend(KeyValueStoreBackend):
     scheme = 'http'
     host = 'localhost'
     port = 9200
+    username = ''
```

Comment:
Perhaps `None` would be easier to detect, since these parameters are optional.

Reply:
Done